### ISSUE TITLE
Rails/RelativeDateConstant fails on constant destructuring

### ISSUE THREAD

**Issue by blimmer** [bbatsov/rubocop#4596]
## Expected behavior

No error thrown from this cop.

## Actual behavior

Errors thrown, with instructions to report.

## Steps to reproduce the problem

create this file:

```ruby
class Foo
  STARTED, COMPLETED, FAILED = %w(started completed failed)
end
```

and run `Rails/RelativeDateConstant` cop.

You get this error output:

```bash
An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:4.
An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:13.
For /Users/blimmer/code/example: configuration from /Users/blimmer/code/example/.rubocop.yml
Inheriting configuration from /Users/blimmer/code/example/.rubocop_todo.yml
Default configuration from /Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/config/default.yml
Inheriting configuration from /Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/config/enabled.yml
Inheriting configuration from /Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/config/disabled.yml
Inspecting 1 file
Scanning /Users/blimmer/code/example/example.rb
undefined method `lambda_or_proc?' for nil:NilClass
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/relative_date_constant.rb:30:in `on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:43:in `block (2 levels) in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:106:in `with_cop_error_handling'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:42:in `block in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `on_casgn'
(eval):2:in `block in on_mlhs'
(eval):2:in `each'
(eval):2:in `on_mlhs'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_mlhs'
(eval):2:in `block in on_masgn'
(eval):2:in `each'
(eval):2:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:88:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:12:in `walk'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:60:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:124:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:112:in `offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:54:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:248:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:195:in `block in do_inspection_loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:227:in `block in iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:191:in `do_inspection_loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:101:in `block in file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:111:in `file_offense_cache'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:99:in `file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:90:in `process_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:68:in `block in each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `reduce'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:57:in `inspect_files'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:36:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:72:in `execute_runner'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:27:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:13:in `block in <top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:12:in `<top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `load'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `<main>'
undefined method `lambda_or_proc?' for nil:NilClass
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/relative_date_constant.rb:30:in `on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:43:in `block (2 levels) in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:106:in `with_cop_error_handling'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:42:in `block in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `on_casgn'
(eval):2:in `block in on_mlhs'
(eval):2:in `each'
(eval):2:in `on_mlhs'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_mlhs'
(eval):2:in `block in on_masgn'
(eval):2:in `each'
(eval):2:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:88:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:12:in `walk'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:60:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:124:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:112:in `offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:54:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:248:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:195:in `block in do_inspection_loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:227:in `block in iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:191:in `do_inspection_loop'An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:24.

3 errors occurred:
An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:4.
An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:13.
An error occurred while Rails/RelativeDateConstant cop was inspecting /Users/blimmer/code/example/example.rb:2:24.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.48.1 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)

/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:101:in `block in file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:111:in `file_offense_cache'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:99:in `file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:90:in `process_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:68:in `block in each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `reduce'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:57:in `inspect_files'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:36:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:72:in `execute_runner'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:27:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:13:in `block in <top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:12:in `<top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `load'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `<main>'
undefined method `lambda_or_proc?' for nil:NilClass
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/rails/relative_date_constant.rb:30:in `on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:43:in `block (2 levels) in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:106:in `with_cop_error_handling'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:42:in `block in on_casgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:41:in `on_casgn'
(eval):2:in `block in on_mlhs'
(eval):2:in `each'
(eval):2:in `on_mlhs'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_mlhs'
(eval):2:in `block in on_masgn'
(eval):2:in `each'
(eval):2:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_masgn'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:88:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:47:in `on_class'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/ast/traversal.rb:12:in `walk'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/commissioner.rb:60:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:124:in `investigate'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:112:in `offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cop/team.rb:54:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:248:in `inspect_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:195:in `block in do_inspection_loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:227:in `block in iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:220:in `iterate_until_no_changes'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:191:in `do_inspection_loop'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:101:in `block in file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:111:in `file_offense_cache'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:99:in `file_offenses'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:90:in `process_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:68:in `block in each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `reduce'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:65:in `each_inspected_file'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:57:in `inspect_files'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/runner.rb:36:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:72:in `execute_runner'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/lib/rubocop/cli.rb:27:in `run'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:13:in `block in <top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/Users/blimmer/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rubocop-0.48.1/bin/rubocop:12:in `<top (required)>'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `load'
/Users/blimmer/.rbenv/versions/2.4.1/bin/rubocop:22:in `<main>'
C

Offenses:

example.rb:2:5: C: Style/ParallelAssignment: Do not use parallel assignment.
    STARTED, COMPLETED, FAILED = %w(started completed failed)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
Finished in 0.11080699996091425 seconds
```

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by blimmer** [comment]
Sorry this is fixed on 49.1.
